sqlmesh/core/model/kind.py

```diff
@@ -1004,6 +1053,15 @@ def create_model_kind(v: t.Any, dialect: str, defaults: t.Dict[str, t.Any]) -> M
         ):
             props["on_destructive_change"] = defaults.get("on_destructive_change")
 
+        # only pass the on_additive_change user default to models inheriting from _Incremental
+        # that don't explicitly set it in the model definition
+        if (
```

Comment:
Any chance we need to duplicate this entire block instead of doing something like:
```
if issubclass(kind_type, _Incremental):
    for on_change_property in ("on_additive_change", "on_desctructive_change"):
        ...
```